post_install.template

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+BREW_HOME="${0%/*}"
+[[ -x $BREW_HOME/bin/brew ]] || \
+  (echo "This script should be located inside Linuxbrew directory."; exit 1)
+
+ORIG_BREW_HOME="_ORIG_BREW_HOME_"
+ORIG_LEN=${#ORIG_BREW_HOME}
+
+BREW_HOME="$PWD"
+LEN=${#BREW_HOME}
+[[ $LEN -le $ORIG_LEN ]] || (echo "Linuxbrew absolute path should be no more than $ORIG_LEN \
+bytes, but actual length is $LEN bytes: $BREW_HOME"; exit 1)
+
+BREW_LINK="$(echo "$BREW_HOME-$(head -c $ORIG_LEN </dev/zero | tr '\0' x)" | \
+  cut -c-$ORIG_LEN)"
```

Comment:
Can we use `common.sh` here as well if we install it into the Linuxbrew directory?

Also outer double quotes not needed.